src/editor-plugin/plugin.js

```diff
@@ -150,6 +158,26 @@ const EditorPlugin = () => {
 				} }
 				className="activitypub-visibility"
 			/>
+
+			<BaseControl
+				label={ __( 'Interaction Policy', 'activitypub' ) }
+				help={ __( 'Choose who can interact with this post.', 'activitypub' ) }
+			></BaseControl>
+
+			<SelectControl
+				label={ __( 'Who can quote', 'activitypub' ) }
```

Comment:
```suggestion
			<SelectControl
				label={ __( 'Who can quote this post?', 'activitypub' ) }
				help={ __( 'Quoting allows others to reshare your post while adding their own commentary.', 'activitypub' ) }
```

The two controls attempting to create hierarchy feel a bit awkward. Could we try playing with label and help in the select control to create the necessary context for this to work?

Reply:
I am also not very happy with it. I tried a lot of different versions, but hoped this one would be the best compromise.

The idea behind that was, that there might be other "Interaction Policy" controls coming, so it might be nice to somehow group the.